.vscode/settings.json

```diff
@@ -20,5 +20,6 @@
     "**/dist": true,
     "node_modules/**": true,
     "packages/node_modules/**": false
-  }
+  },
+  "files.eol": "\n"
```

Comment:
On windows machines CRLF was used as the eol which would not pass the elsint tests. Setting this to "\n" ensures that no matter what OS is used (in MS Code at least) the eol will be LF and will pass eslint tests